pkg/sbom/cyclonedx/marshal.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 const (
-	Namespace = "aquasecurity:trivy:"
+	Namespace = "Deepfactor:dfctl:"
```

Comment:
Should this not be Deepfactor:dfscan ?

Reply:
Since dfctl is what customers are exposed to, Vikas  decided to keep it dfctl.